### PR TITLE
✨  FWF:3372 [Feature] Added keycloak data package

### PR DESCRIPTION
**Pull request by shuhaib-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3372
Issue Type: BUG/ FEATURE

# Changes
1. Deleted Old Dockerfile
2. Added CD for pushing package image

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

### CODE REVIEW

File: forms-flow-idm/keycloak/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,9 @@
-FROM jboss/keycloak
+FROM formsflow/forms-flow-idm-theme
 
-RUN sed -i -E "s/(<staticMaxAge>)2592000(<\/staticMaxAge>)/\1\-1\2/" /opt/jboss/keycloak/standalone/configuration/standalone.xml
-RUN sed -i -E "s/(<cacheThemes>)true(<\/cacheThemes>)/\1false\2/" /opt/jboss/keycloak/standalone/configuration/standalone.xml
-RUN sed -i -E "s/(<cacheTemplates>)true(<\/cacheTemplates>)/\1false\2/" /opt/jboss/keycloak/standalone/configuration/standalone.xml
\ No newline at end of file
+RUN mkdir -p /opt/keycloak/themes/formsflow
```
Comment (amal-k-soman-aot) [inline forms-flow-idm/keycloak/Dockerfile:3]: @shuhaib-aot please update the keycloak path ith Docker file like /opt/bitnami/keycloak/themes/formsflow

File: forms-flow-idm/keycloak/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,9 @@
-FROM jboss/keycloak
+FROM formsflow/forms-flow-idm-theme
```
Comment (amal-k-soman-aot) [inline forms-flow-idm/keycloak/Dockerfile:1]: change image version to formsflow/forms-flow-idm-theme:v7.0

File: forms-flow-idm/keycloak/Dockerfile (hunk 1 above)
Comment (shuhaib-aot) [reply forms-flow-idm/keycloak/Dockerfile]: @amal-k-soman-aot  resolved please check

File: forms-flow-idm/keycloak/Dockerfile (hunk 2 above)
Comment (shuhaib-aot) [reply forms-flow-idm/keycloak/Dockerfile]: @amal-k-soman-aot  please check

**Review by abilpraju-aot — APPROVED** [review]

**Review by amal-k-soman-aot — APPROVED** [review]

**Pull request merged**